CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 Enterprise:
 * Fixed a redirection issue with the SAML IdP authentication middleware which prevented users from signing into the service provider when an SAML authentication request was made with an HTTP-POST binding protocol, and user's didn't already have an active session with Teleport. [#4806](https://github.com/gravitational/teleport.e/pull/4806)
 * SAML applications can now be deleted from the Web UI. [#4778](https://github.com/gravitational/teleport.e/pull/4778)
+* Fixed `tbot` FIPS builds failing to start due to missing boringcrypto dependency [#4757](https://github.com/gravitational/teleport.e/pull/4757)
```

Comment:
Can we mention the first release that had the issue? Something like "fixed an issue introduced in Teleport X.Y.Z that resulted in FIPS builds failing to start..."

Reply:
Sure, done